pegasus/test/test_dev_routes.rb

```diff
@@ -167,4 +168,49 @@ def in_rack_env(env)
       CDO.rack_env = original_rack_env
     end
   end
+
+  describe 'api/dev/set-last-dtt-green' do
+    before do
+      $log.level = Logger::ERROR
+
+      CDO.slack_start_build_token = FAKE_SLACK_SLASH_TOKEN
+
+      @default_param = {
+        token: FAKE_SLACK_SLASH_TOKEN,
+        user_name: 'turing'
+      }
+    end
+
+    def in_rack_env(env)
```

Comment:
The `in_rack_env` and `make_test_pegasus` methods are being shamelessly copied from above. As noted in the PR conversation, I intend to follow up with a PR that DRYs up this whole test suite.